params/protocol_params.go

```diff
@@ -188,17 +190,20 @@ const (
 )
 
 const (
-	DefaultBlockGenerationInterval    = int64(1) // unit: seconds
-	DefaultBlockGenerationTimeLimit   = 250 * time.Millisecond
-	DefaultOpcodeComputationCostLimit = uint64(100000000)
+	DefaultBlockGenerationInterval  = int64(1) // unit: seconds
+	DefaultBlockGenerationTimeLimit = 250 * time.Millisecond
+
+	OpcodeComputationCostLimitInfinite = math.MaxUint64
+	OpcodeComputationCostLimitOverride = uint64(0)
```

Comment:
```suggestion
```

Reply:
Moved to computation_cost_params.go https://github.com/klaytn/klaytn/commit/3709fd8a22b92ef8553e5d422e216d8fc0436e99